<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerdialog</class>
 <widget class="QDialog" name="registerdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="registerBtn">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>272</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>182</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddressEdit">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>212</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="ContactEdit">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>242</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="WindowName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>### Register User Window</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>## Hello User</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>182</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name - </string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>211</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Address - </string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::MarkdownText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Contect -</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::MarkdownText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>registerBtn</sender>
   <signal>accepted()</signal>
   <receiver>registerdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>registerBtn</sender>
   <signal>rejected()</signal>
   <receiver>registerdialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
